src/Args.php

```diff
@@ -181,11 +181,7 @@ public function getOpt($option, $default = null) {
      * @return Args Returns $this for fluent calls.
      */
     public function setOpt($option, $value) {
-        if ($value === null) {
-            unset($this->opts[$option]);
-        } else {
-            $this->opts[$option] = $value;
-        }
+        $this->opts[$option] = $value;
```

Comment:
Yeah. $value can be null now.

I guess I could make some changes to the Cli::parseRaw() function to set the default to '' instead of null.